ExampleCodes/EB/STLtest/inputs

```diff
@@ -1,6 +1,13 @@
-prob_lo=  0   0  0
-prob_hi=  220 35 60
-ncells =  128 32 64
-stl_file=airfoil.stl
-max_grid_size=8
-outside_point=250.0 250.0 250.0
+eb2.geom_type             = stl
+eb2.stl_file              = airfoil.stl
+
+eb2.stl_scale             = 1                   # default is 1
+eb2.stl_center            = 0      0      0     # default is (0,0,0)
+eb2.stl_reverse_normal    = 1                   # default is 0. use "1" to see the volume of the stl object 
```

Comment:
Since this is in amrex-tutorials and this parameter `stl_reverse_normal` is rarely used, we might want to remove this. To "see" the volume of the stl object, we could make a new MultiFab in the code to store `1-volfrac`.

Reply:
I edited the main file to draw the stl object. However, I suggest keeping the eb2.stl_reverse_normal option in the input file to inform users that this parameter exists. We have used this in some PeleC test cases.